### PR TITLE
Update plugin to the v2.0 api

### PR DESCRIPTION
**Pull request by ph**

### CODE REVIEW

File: spec/inputs/elasticsearch_spec.rb [hunk 1]
```diff
@@ -208,10 +208,10 @@
           queue.pop
         end
 
-        expect(event[metadata_field]["_index"]).to eq('logstash-2014.10.12')
-        expect(event[metadata_field]["_type"]).to eq('logs')
-        expect(event[metadata_field]["_id"]).to eq('C5b2xLQwTZa76jBmHIbwHQ')
-        expect(event[metadata_field]["awesome"]).to eq("logstash")
+        expect(event.get(metadata_field)["_index"]).to eq('logstash-2014.10.12')
```
Comment (talevy) [inline spec/inputs/elasticsearch_spec.rb]: is this:
```
event.get("[#{metadata_field}][_index]")
```
more proper?

**Comment by ph** [comment]
updated

**Comment by ph** [comment]
@talevy update with your comments

**Comment by talevy** [comment]
LGTM

**Pull request merged**